src/now.js

```diff
@@ -369,14 +369,23 @@ const main = async (argv_) => {
     !ctx.argv.includes('-h') && !ctx.argv.includes('--help') &&
     !ctx.argv.includes('-t') && !ctx.argv.includes('--token')
   ) {
-    console.log(info(`No existing credentials found. Please log in:`))
+    // $FlowFixMe
+    const { isTTY } = process.stdout
 
-    subcommand = 'login'
-    ctx.argv[2] = 'login'
+    if (isTTY) {
+      console.log(info(`No existing credentials found. Please log in:`))
 
-    // Ensure that sub commands lead to login as well, if
-    // no credentials are defined
-    ctx.argv = ctx.argv.splice(0, 3)
+      subcommand = 'login'
+      ctx.argv[2] = 'login'
+
+      // Ensure that sub commands lead to login as well, if
+      // no credentials are defined
+      ctx.argv = ctx.argv.splice(0, 3)
+    } else {
+      console.log(error(`No existing credentials found. Please ` +
+      `use ${chalk.grey('`now login`')} to log in or pass ${chalk.grey('`--token`')}`))
```

Comment:
I'm not so sure if coloring always work on non-tty.

Reply:
✅ Fixed